pkg/services/alerting/notifiers/googlechat.go

```diff
@@ -16,16 +16,16 @@ import (
 func init() {
 	alerting.RegisterNotifier(&alerting.NotifierPlugin{
 		Type:        "googlechat",
-		Name:        "Google Hangouts Chat",
-		Description: "Sends notifications to Google Hangouts Chat via webhooks based on the official JSON message format",
+		Name:        "Google Chat",
```

Comment:
Hi! :wave: I'm afraid we are no longer accepting changes for Legacy Alerting as it's deprecated as of Grafana 10 and is planned to be removed in Grafana 11. However, we would love to see this change for our new Unified Alerting product. You can find the Google Chat integration [here](https://github.com/grafana/alerting/tree/main/receivers/googlechat).

Reply:
@grobinson-grafana Since this is an external contribution it might be better if someone in the alerting squad make those changes to this PR.

Reply:
I've made a ticket to track this https://github.com/grafana/alerting/issues/121.